src/anm/media/video.js

```diff
@@ -137,7 +137,7 @@ Video.prototype.load = function(elm, player) {
 };
 /** @private @method apply */
 Video.prototype.apply = function(ctx) {
-    if (this.video) ctx.drawImage(this.video, 0, 0, this.video.width, this.video.height, 0, 0, this.size[0], this.size[1]);
+    if (this.video) ctx.drawImage(this.video, 0, 0, this.video.videoWidth, this.video.videoHeight, 0, 0, this.size[0], this.size[1]);
```

Comment:
yep, that's correct fix, at least those are always be set if the video is loaded.

I was contemplating what to use in the editor: width or videoWidth and for whatever reason chose width, although videoWidth is better